content/kubernetes/security/ldap.md

```diff
@@ -0,0 +1,158 @@
+---
+Title: Enable LDAP authentication
+linkTitle: Enable LDAP
+description: Enable LDAP authentication for Redis Enterprise for Kubernetes. 
+weight: 95
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [ 
+  /kubernetes/security/ldap.md,
+  /kubernetes/security/ldap/,
+]
+---
+
+## LDAP support for Redis Enterprise Software
+
+Redis Enterprise Software supports LDAP authentication and authorization through [role-based access controls]({{<relref "/rs/security/access-control/">}}) (RBAC). You can map LDAP groups to {[Redis Enterprise roles]({{<relref "/rs/security/access-control/create-roles.md">}}) to control access to your database and the [admin console]({{<relref "/rs/security/admin-console-security/">}}). For more details on how LDAP works with Redis Enterprise, see [LDAP authentication]({{<relref "/rs/security/access-control/ldap/">}}).
+
+Redis Enterprise for Kubernetes supports enabling and configuring LDAP authentication using the RedisEnterpriseCluster (REC) custom resource. Currently, the Redis Enterprise cluster (REC) only supports configuration related to the LDAP server, such as server addresses, connection details, credentials, and query configuration.
+
+To [map LDAP groups to Redis Enterprise access control roles]({{<relref "/rs/security/access-control/ldap/enable-role-based-ldap.md">}}), you'll need to use the Redis Enterprise [API]({{<relref "/rs/references/rest-api/requests/ldap_mappings/">}}) or [admin console]({{<relref "/rs/security/access-control/ldap/enable-role-based-ldap.md">}}).
+
+## Enable LDAP for Redis Enterprise cluster (REC)
+
+Use the `.spec.ldap` field in the `RedisEnterpriseCluster` custom resource to enable LDAP for Redis Enterprise cluster (REC).
+
+The following `RedisEnterpriseCluster` example resource enables a basic LDAP configuration:
```

Comment:
```suggestion
"Follow this example to enable a basic LDAP configuration:"
```

Reply:
I'm not sure I want to suggest they use they verbatim. It's just an example of the syntax but they need to substitute their own values for some of the lines. In the steps below we lay out which values they need to supply.